projects/1.18/assets/create-cafe/createcafe/lang/zh_cn.json

```
@@ -1 +1,171 @@
+    "item.createcafe.yucca_tea_bucket": "丝兰奶茶桶",
+    
+    "item.createcafe.coffee_beans": "咖啡豆",
+    
+    "item.createcafe.cassava_seeds": "木薯种子",
+  
+    "item.createcafe.mana_berries": "蓝莓",
+    "item.createcafe.oreo": "奥利奥",
+    "item.createcafe.oreo_dough": "奥利奥面团",
+    "item.createcafe.oreo_half": "半块奥利奥",
+    "item.createcafe.oreo_incomplete": "未加工的奥利奥",
+    "item.createcafe.oreo_half_raw": "半块奥利奥生面团",
+    "item.createcafe.oreo_crushed": "压碎的奥利奥",
+    "item.createcafe.blood_orange": "血橙",
+  
+    "item.createcafe.empty_boba_cup": "空的奶茶杯",
+    "item.createcafe.boba_cup": "有珍珠的奶茶杯",
+    "item.createcafe.boba": "珍珠",
+    "item.createcafe.raw_boba": "未加工的珍珠",
+    "item.createcafe.tapioca_flour": "木薯淀粉",
+  
+    "block.createcafe.cassava_crop": "木薯种子",
+    "item.createcafe.cassava_root": "木薯根部",
+  
+    "block.createcafe.coffee_crop": "咖啡豆",
+    "item.createcafe.roasted_coffee": "烤咖啡豆",
+    "item.createcafe.coffee_fruit": "咖啡果",
+  
+    "item.createcafe.mango_milk_tea": "芒果奶茶",
+    "block.createcafe.mango_tea": "芒果奶茶",
+    "item.createcafe.lime_milk_tea": "酸橙奶茶",
+    "block.createcafe.lime_tea": "酸橙奶茶",
+    "item.createcafe.strawberry_milk_tea": "草莓奶茶",
+    "block.createcafe.strawberry_tea": "草莓奶茶",
+    "item.createcafe.peach_milk_tea": "桃子奶茶",
+    "block.createcafe.peach_tea": "桃子奶茶",
+    "item.createcafe.plum_milk_tea": "李子奶茶",
+    "block.createcafe.plum_tea": "李子奶茶",
+    "item.createcafe.jackfruit_milk_tea": "菠萝蜜奶茶",
+    "block.createcafe.jackfruit_tea": "菠萝蜜奶茶",
+    "item.createcafe.apricot_milk_tea": "杏子奶茶",
+    "block.createcafe.apricot_tea": "杏子奶茶",
+    "item.createcafe.durian_milk_tea": "榴莲奶茶",
+    "block.createcafe.durian_tea": "榴莲奶茶",
+    "item.createcafe.fig_milk_tea": "无花果奶茶",
+    "block.createcafe.fig_tea": "无花果奶茶",
+    "item.createcafe.grape_milk_tea": "葡萄奶茶",
+    "block.createcafe.grape_tea": "葡萄奶茶",
+    "item.createcafe.grapefruit_milk_tea": "柚子奶茶",
+    "block.createcafe.grapefruit_tea": "柚子奶茶",
+    "item.createcafe.starfruit_milk_tea": "杨桃奶茶",
+    "block.createcafe.starfruit_tea": "杨桃奶茶",
+    "item.createcafe.blood_orange_milk_tea": "血橙奶茶",
+    "block.createcafe.blood_tea": "血橙奶茶",
+    "item.createcafe.lychee_milk_tea": "荔枝奶茶",
+    "block.createcafe.lychee_tea": "荔枝奶茶",
+    "item.createcafe.blueberry_milk_tea": "蓝莓奶茶",
+    "block.createcafe.blueberry_tea": "蓝莓奶茶",
+    "item.createcafe.pineapple_milk_tea": "菠萝奶茶",
+    "block.createcafe.pineapple_tea": "菠萝奶茶",
+    "item.createcafe.apple_milk_tea": "苹果奶茶",
+    "block.createcafe.apple_tea": "苹果奶茶",
+    "item.createcafe.pumpkin_milk_tea": "南瓜奶茶",
+    "block.createcafe.pumpkin_tea": "南瓜奶茶",
+    "item.createcafe.avocado_milk_tea": "牛油果奶茶",
+    "block.createcafe.avocado_tea": "牛油果奶茶",
+    "item.createcafe.sweetberry_milk_tea": "甜浆果奶茶",
+    "block.createcafe.sweetberry_tea": "甜浆果奶茶",
+    "item.createcafe.tamarind_milk_tea": "罗望子果奶茶",
+    "block.createcafe.tamarind_tea": "罗望子果奶茶",

```

```suggestion
    "item.createcafe.tamarind_milk_tea": "酸角奶茶",
    "block.createcafe.tamarind_tea": "酸角奶茶",
```
https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/pull/1970#discussion_r867725820